### DISCUSSION TITLE
Make the "reading" map in My Profile clickable to display the posts read on the clicked day

### DISCUSSION THREAD

**Discussion by donangel** [dailydotdev/daily#1472] [Feature Requests]
As a user, I would like to be able to click on each one of the dots in the reading map in my profile and see all the posts I read on that day:

<img width="744" alt="image" src="https://github.com/user-attachments/assets/e5c7b318-913b-4018-8d00-29329903153c">

- if I click on September 29, 2024, I should see a list of all 6 articles I read on that date

A nice additional option would be to be able to make that information public, e.g., if I enable the information to be public, anyone who browses my profile should be able to click and see what I read on that day. This setting, of course, will be disabled by default.

**Comment by idoshamun** [comment]
Hi @donangel,

That's a nice suggestion! We'll consider it
